src/kiota/Handlers/KiotaGenerateCommandHandler.cs

```diff
@@ -170,4 +172,8 @@ public override async Task<int> InvokeAsync(InvocationContext context)
         get;
         set;
     }
+    public required Option<bool> DisableSSLValidationOption
```

Comment:
this option should also be present in the show and download commands.

Reply:
added to both show and download commands